doc/filters/trim.rst

```diff
@@ -23,4 +27,7 @@ Arguments
 
 * ``character_mask``: The characters to strip
 
+* ``type``: The default is to strip from the start and the end (`both`), but `left`
+and `right` will strip from either the left side or right side only.
```

Comment:
This line should be indented with two spaces so it is rendered as part of the bullet point instead of a new line. See https://github.com/antonydandrea/Twig/blob/2.x_ltrim_rtrim/doc/filters/trim.rst.